spec/requests/api/v1/articles_spec.rb

```diff
@@ -13,8 +14,98 @@
     allow(FeatureFlag).to receive(:enabled?).with(:api_v1).and_return(true)
   end
 
-  describe "GET /api/articles" do
-    before { article }
+  describe "GET /articles" do
+    before do
+      article.update_columns(organization_id: organization.id)
+    end
+
+    # rubocop:disable RSpec/VariableName
+    path "/api/articles" do
+      get "Published articles" do
+        tags "articles"
+        description "This endpoint allows the client to retrieve a list of articles.
+
+\"Articles\" are all the posts that users create on DEV that typically
+show up in the feed. They can be a blog post, a discussion question,
+a help thread etc. but is referred to as article within the code.
+
+By default it will return featured, published articles ordered
+by descending popularity.
+
+It supports pagination, each page will contain `30` articles by default."
+        operationId "getArticles"
+        produces "application/json"
+        parameter "$ref": "#/components/parameters/pageParam"
+        parameter "$ref": "#/components/parameters/perPageParam30to1000"
+        parameter name: :tag, in: :query, required: false,
```

Comment:
These parameter definitions were not extracted as they only appear in this single API call in the V0 docs.